dev/benchmark/add-index-with-load.md

```diff
@@ -0,0 +1,347 @@
+| 16    | 39 |   149.3 |           2893 |
+| 32    | 36 |   113.5 |           2268 |
+| 48    | 33 |   86.2 |           1715 |
+
+![add-index-load-1-b256](/media/add-index-load-1-b256.png)
+
+#### `tidb_ddl_reorg_batch_size = 512`
+
+| tidb_ddl_reorg_worker_cnt |  add_index_durations(s) | sysbench TPS   | sysbench QPS |
+| :------------------------ | :---------------------- | :------------- | :----------- |
+| 1     | 135 |  257.8 |           5147 |
+| 2     | 78 |  252.8 |           5053 |
+| 4     | 49 |  222.7 |           4478 |
+| 8     | 36 |  145.4 |           2904 |
+| 16    | 33 |   109 |           2190 |
+| 32    | 33 |   72.5 |           1503 |
+| 48    | 33 |   54.2 |           1318 |
+
+![add-index-load-1-b512](/media/add-index-load-1-b512.png)
+
+#### `tidb_ddl_reorg_batch_size = 1024`
+
+| tidb_ddl_reorg_worker_cnt |  add_index_durations(s) | sysbench TPS   | sysbench QPS |
+| :------------------------ | :---------------------- | :------------- | :----------- |
+| 1     | 111 |  244.3 |           4885 |
+| 2     | 78 |  228.4 |           4573 |
+| 4     | 54 |  168.8 |           3320 |
+| 8     | 39 |  123.8 |           2475 |
+| 16    | 36 |   59.6 |           1213 |
+| 32    | 42 |   93.2 |           1835 |
+| 48    | 51 |   115.7 |           2261 |
+
+![add-index-load-1-b1024](/media/add-index-load-1-b1024.png)
+
+#### `tidb_ddl_reorg_batch_size = 2048`
+
+| tidb_ddl_reorg_worker_cnt |  add_index_durations(s) | sysbench TPS   | sysbench QPS |
+| :------------------------ | :---------------------- | :------------- | :----------- |
+| 1     | 918 |  243.3 |           4855 |
+| 2     | 1160 |  209.9 |           4194 |
+| 4     | 342 |  185.4 |           3707 |
+| 8     | 1316 |  151.0 |           3027 |
+| 16    | 795 |   30.5 |           679 |
+| 32    | 1130 |   26.69 |           547 |
+| 48    | 893 |   27.5 |           552 |
+
+![add-index-load-1-b2048](/media/add-index-load-1-b2048.png)
+
+#### `tidb_ddl_reorg_batch_size = 4096`
+
+| tidb_ddl_reorg_worker_cnt |  add_index_durations(s) | sysbench TPS   | sysbench QPS |
+| :------------------------ | :---------------------- | :------------- | :----------- |
+| 1     | 3042 |  200.0 |           4001 |
+| 2     | 3022 |  203.8 |           4076 |
+| 4     | 858 |  195.5 |            3971 |
+| 8     | 3015 |  177.1 |           3522 |
+| 16    | 837 |   143.8 |           2875 |
+| 32    | 942 |   114 |           2267 |
+| 48    | 187 |   54.2 |           1416 |
+
+![add-index-load-1-b4096](/media/add-index-load-1-b4096.png)
+
+### Test conclusion
+
+When you perform frequent write operations (this test involves `UPDATE`, `INSERT` and `DELETE` operations) to the target column of `Add Index` statement, the default `Add Index` configuration has a significant impact on the online workload of the system. It is mainly because of the write conflicts caused by the concurrent `Add Index` operation and column update. The performance of the system is as follows:
```

Comment:
```suggestion
When you perform frequent write operations (this test involves `UPDATE`, `INSERT` and `DELETE` operations) to the target column of the `Add Index` statement, the default `Add Index` configuration has a significant impact on the online workload of the system. It is mainly because of the write conflicts caused by the concurrent `Add Index` operation and column update. The performance of the system is as follows:
```